src/appleseed/foundation/image/genericimagefilewriter.cpp

```diff
@@ -57,6 +60,13 @@ struct GenericImageFileWriter::Impl
 {
     std::vector<const ICanvas*>     m_canvas;
     std::vector<OIIO::ImageSpec>    m_spec;
+
+#ifdef OIIO_VERSION >= 20000
+    std::unique_ptr<OIIO::ImageOutput>  m_writer;
```

Comment:
Align members.

Reply:
I would also suggest ordering them similarly to how they are initialized, which seems to be: filename, writer, canvas, spec.